kayenta-signalfx/kayenta-signalfx.gradle

```diff
@@ -109,21 +109,16 @@ integrationTest {
 }
 
 dependencies {
-  compile project(":kayenta-core")
+  implementation project(":kayenta-core")
 
-  compile spinnaker.dependency('bootWeb')
-  compile spinnaker.dependency("korkSwagger")
-  compile spinnaker.dependency('lombok')
+  implementation 'com.signalfx.public:signalfx-java:0.0.48'
 
-  compile "com.netflix.spinnaker.orca:orca-core:$orcaVersion"
-
-  compile group: 'com.signalfx.public', name: 'signalfx-java', version: '0.0.48'
-
-  testCompile group: 'com.tngtech.java', name: 'junit-dataprovider', version: '1.13.1'
+  testImplementation 'com.tngtech.java:junit-dataprovider:1.13.1'
+  testImplementation project(":kayenta-standalone-canary-analysis")
 
   // Integration Test dependencies
   integrationTestCompile sourceSets.main.output
```

Comment:
I had to add the following additional lines to run the integration tests

```groovy
  integrationTestAnnotationProcessor platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
  integrationTestAnnotationProcessor "org.projectlombok:lombok"
  integrationTestCompileOnly "org.projectlombok:lombok"
```

Reply:
oh thanks, I'll update that